modules/discover/con-supported-platforms.adoc

```diff
@@ -0,0 +1,14 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-11-08
+
+:_mod-docs-content-type: CONCEPT
+
+[id="supported-platforms_{context}"]
+== Supported platforms
+
+{product} runs on {ocp-short} {ocp-version-min}-{ocp-version} on supported architectures.
+
+
+.Additional resources
+
+* For more information about the different supported hosted environments, platforms and versions, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/installing_red_hat_developer_hub_on_openshift_container_platform/index#assembly-install-rhdh-ocp-operator[Supported platforms].
```

Comment:
Consider removing this link, and moving the information from there to here.

```suggestion
```

Reply:
... which means linking from the install guides (plural) to the supported platform doc (centralized).